.flake8

```diff
@@ -0,0 +1,47 @@
+[flake8]
+################### FILE PATTERNS ##########################
+
+# Provide a comma-separated list of glob patterns to exclude from checks.
+exclude =
+    .git,
+    __pycache__,
+    .pytest_cache,
+    .mypy_cache,
+    .venv,
+    .vscode,
+    .idea,
+    test
+# Provide a comma-separate list of glob patterns to include for checks.
+filename =
+    *.py
+
+
+################### LINTING ################################
+# Report all errors, even if it is on the same line as a `# NOQA` comment.
+disable-noqa = False
+
+# Set the maximum length that any line (with some exceptions) may be.
+max-line-length = 120
```

Comment:
this line length conflicts with https://github.com/ebi-ait/ingest-broker/blob/f09b172e2b0a55a983952894c620b9f7cefc511e/.pylintrc#L24